TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -396,62 +395,63 @@ public class RoutingManager
             ExtSegment prevSeg = Constants.ManagerFactory.ExtSegmentManager.ExtSegments[segmentId];
             ExtSegmentEnd prevEnd = segEndMan.ExtSegmentEnds[segEndMan.GetIndex(segmentId, startNode)];
 
-            ushort prevSegmentId = segmentId;
-            int prevLaneIndex = laneIndex;
+            ushort prevSegmentId = segmentId; // this variable is duplicate (prevSegmentId == segmentId always)
+            int prevLaneIndex = laneIndex; // this variable is duplicate (prevLaneIndex == laneIndex always)
             uint prevLaneId = laneId; // this variable is duplicate (prevLaneId == laneId always)
```

Comment:
is there any better way of dealing with these duplicate variables?
is there any way of marking that as read-only or something?
should I change parameter names or would that make the method confusing?

Reply:
I just renamed parameters. we have enough variables in this method. we don't need duplicates!